sig-and-wg/wg-prow/threat-modelling.md

```diff
@@ -0,0 +1,43 @@
+# About
+Here the results of the first threat modelling are documented.
+
+The treat modelling was done in October 2018 in three sessions with Suleyman Akbas, Andreas Thaler and a representive of the kyma security team.
+
+# What was done:
+Bringing the different systems into one picture
+Discussed every communication channel
+Identified the assets being worth to protect
+Identified the secrets in the game
+
+# Assumptions
+There is one prow cluster only. If there are more installations they need to be protected in a similar way.
+
+# Working picture
```

Comment:
Pls add an introductory sentence: `The diagram presents an overview of the Prow architecture:`

Reply:
done